templates/epic.xml.jinja2

```diff
@@ -46,6 +46,8 @@
     </documentation>
     <include ref="${DETECTOR_PATH}/compact/far_forward/fields_{{ pbeam | default("275", true) }}.xml" />
     <include ref="${DETECTOR_PATH}/compact/far_forward/definitions.xml" />
+    <include ref="${DETECTOR_PATH}/compact/far_backward/fields_{{ ebeam | default("18", true) }}.xml" />
```

Comment:
Do the fields only depend on the electron beam energy?

Reply:
From the tables here:
https://indico.bnl.gov/event/10974/contributions/51260/

The fields as I understand them are just scaling with the electron energy to the first few decimal places, however there is variation on the same level when the ion beam energy is changed. If you know where a more up to date set of values it would be useful to check. I imagine these will be tweaked in the future to not just be simply scalable.

Reply:
I mean that at this point you add fields_18.xml etc. But won't we need fields_18x275.xml at some point?

Reply:
Good question, I'm not sure, the focussing might be changed slightly? Isn't it more likely the solenoid will be tuned for different energy physics.

Reply:
No idea... @ajentsch How much will proton beam energy affect (far) forward beam magnet settings?

Reply:
So, the magnet settings mostly scale with energy, except the B0pf (pf = proton forward) magnet, which stays constant at all beam settings.

However, this may not always be true long-term (there are changes to the lattice on the horizon for the hadron beam). 

In other words, we should avoid introducing scale factors as this stage of the games, and use the exact values from the tables (e.g. separate XML for various beam settings) to avoid possible pitfalls down the road.

If the concern is figuring out how to change the fields at runtime for the user, that's something we need to think about.

Reply:
The outgoing electron magnets don't currently have different proposed fields dependant on the proton energy from some second order focussing? For the time being we could keep the forward and backwards settings in separate files rather than making one for each combination?

Reply:
@simonge No, what you have is the current official set of tables, and what everyone should be using until there is an official new release.

Reply:
So you don't think we need something like this, for future proofness (also for far_forward)?
```suggestion
    <include ref="${DETECTOR_PATH}/compact/far_backward/fields_{{ ebeam | default("18", true) }}x{{ pbeam | default("275", true) }}.xml" />
```

Reply:
They might then be better consolidated into one file in the compact/fields folder. It is also quite likely in the future the simple dipole and quadrupole fields should be replaced with maps. I don't think either implementation at this point would cause any significant difference in the work needed to make changes down the line.